scripts/send-subscription-receipts.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+require('habitat').load();
+
+var stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
+var hatchet = require('hatchet');
+var async = require('async');
+var moment = require('moment');
+
+var stripe_charge_list_opts = {
+  created: {
+    lte: moment.unix(new Date(process.env.BEFORE_DATE)).valueOf()
```

Comment:
This needs to be converted to a regular Unix epoch count in seconds, not the Javascript variant that counts in ms. You could remove the moment dependency and change it to:

`new Date(process.env.BEFORE_DATE).valueOf() / 1000`